File: omni.java
public class omni {
   public static void main(String[] args) {
      System.out.println(parent());
   }

   public static int add(int x, int y) {
      return x + y;
   }

   public static int mul(int x, int y) {
      return x * y;
   }

   public static int parent() {
      int a = 0;
      int x = 200;
      int y = 10;
      boolean z = true; 
      if (z) {
         a = add(x,y);
         a += 1;
      }
      return a;
   }
}

File: omni.py
def main():
	print(coolMath())

def add(x,y):
	return x + y

def mul(x,y):
	return x * y

def coolMath():
	a = 0
	x = 200
	y = 10
	z = True
	if z:
		a = add(x,y)
		a += 1
	return a

main()
